src/systems/physics-system.js

```diff
@@ -155,9 +153,10 @@ export class PhysicsSystem {
         if (this.objectMatricesFloatArray.buffer.byteLength !== 0) {
           for (let i = 0; i < this.bodyUuids.length; i++) {
             const uuid = this.bodyUuids[i];
-            const index = this.uuidToIndex[uuid];
-            const type = this.bodyOptions[uuid].type ? this.bodyOptions[uuid].type : TYPE.DYNAMIC;
-            const object3D = this.object3Ds[uuid];
+            const body = this.bodies.get(uuid);
```

Comment:
maybe `bodyInfo` or `infoForBody` or something that indicates that this is a collection of various stuff associated with this body uuid